stubs/erlang-ssl/run

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env escript
+%% -*- erlang -*-
+ 
+%% trytls (https://github.com/ouspg/trytls) stub for erlang
+ 
+%% Uncomment the following line if needed. Hard to say if compiling is worthwhile here.
+%% -mode(compile).
+ 
+ 
+ 
+main([Url, Port]) ->
+    start_deps(),
+    Result = ssl:connect(Url, Port, []),
+    check(Result);
+ 
+main([Url, Port, CaFile]) ->
+    start_deps(),
+    Result = ssl:connect(Url, Port, [{cacertfile, CaFile}]),
+    check(Result);
+ 
+main(_) ->
+    usage().
+ 
+usage() ->
+    io:format("usage: run <domain> <port> [ca_file]~n"),
+    io:format("Will do request as: https://<domain>:<port>~n").
```

Comment:
`"Will do request as: https://<domain>:<port>~n"` is probably not correct for the erlang-ssl stub.